doc/api/fs.md

```diff
@@ -290,6 +290,15 @@ added: v0.1.93
 
 Emitted when the ReadStream's file is opened.
 
+### Event: 'ready'
+<!-- YAML
+added: v9.11.0
+-->
+
+Emitted when the ReadStream's file is ready.
```

Comment:
Nit: `ReadStream's` -> `` `ReadStream`'s ``

Reply:
Hi! Does this mean the rest of the docs should comply to that? I'm seeing that the `'open'` event doesn't wrap `ReadStream` in backticks either. Maybe a new issue should be opened to fix the various inconsistencies instead?

Reply:
Our docs have many format inconsistencies, it may be worth to not add more in new changes. But I may be wrong, so you can decide at your own discretion)

Reply:
I've searched all the docs for this pattern and found only these few fragments. Thank you for reporting, I will fix them ASAP: https://github.com/nodejs/node/pull/19938

Reply:
Fixed.